protocol/messenger.go

```diff
@@ -521,7 +521,17 @@ func NewMessenger(
 			pushNotificationClient.Stop,
 			communitiesManager.Stop,
 			encryptionProtocol.Stop,
-			transp.ResetFilters,
+			func() error {
+				ctx, cancel := context.WithTimeout(context.Background(), time.Second*1)
```

Comment:
Nothing major, but you don't need to multiply by 1 🤷 

```suggestion
				ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Reply:
ouch :)